File: src/main/java/CacheService.java
import java.time.LocalDate;
import org.springframework.stereotype.Service;

@Service
public class CacheService {
    private Cache cache=new Cache(LocalDate.now(), LocalDate.now().minusDays(1000));

    public void setDay(LocalDate day) {
        cache.setDay(day);
    }

    public void getDay(LocalDate day) {
        cache.setDay(day);
    }
}

File: src/main/kotlin/com/example/bitsetservice/CacheService.kt
package com.example.bitsetservice

import org.springframework.stereotype.Service
import java.time.LocalDate

@Service
class CacheService {
    private val cache = Cache(LocalDate.now().minusDays(10000), LocalDate.now())
    fun setDay(day: LocalDate) {
        cache.setDay(day)
    }

    fun getDay(day: LocalDate) : Boolean {
        return cache.getDay(day)
    }

    fun getCache() : Cache {
        return cache
    }
}